Minion id is visible in System Info box (#438)
SUSE/doc-susemanager: modules/reference/pages/systems/system-details.adoc

IPv6 Address:::
The IPv6 address of the client.

Minion Id:::

> If the client is a salt minion, the minion id value is shown.

Reviewer: ```suggestion
For salt clients, the minion identification value is shown.
```

> If the client is a salt minion, the minion id value is shown.

Reviewer: ```suggestion
On salt clients only, shows the minion identification value.
```

Author: That's the difference outcome when *you* are a mother language and a great writer......but I don't!! :wink:

Reviewer: I still love your work, Dario <3
